src/components/screens/ObjectScreen.js

```diff
@@ -3,18 +3,45 @@
 import React, { Component } from "react";
 import { connect } from "react-redux";
 import ObjectView from "../shared/ObjectView";
+import LangService from "../../services/langService";
 import { AnalyticsUtils } from "../../utils/";
+
+import fetchService from "../../services/FetchService";
+import MediaService from "../../services/mediaService";
```

Comment:
It would be really nice to get the MediaService away from this component, if possible.
So the middleware is the only one that interacts directly with the MediaService.

Reply:
✅  done